### ISSUE TITLE
mts command

### ISSUE THREAD

**Issue by samine66m** [hfawaz/dl-4-tsc#1]
Can you please share the command to run a mts dataset?

**Comment by hfawaz** [comment]
For MTS datasets, 

You should first download the MTS archive from this [link](http://www.mustafabaydogan.com/files/viewcategory/20-data-sets.html). 

Then you should convert these to the UCR format using [this function](https://github.com/hfawaz/dl-4-tsc/blob/4cb3bf7387b80e768b88d5b44047d239844a41ac/utils/utils.py#L149). 

Finally you should issue the following command for example: ```python3 main.py mts_archive Libras fcn _itr_8``` to run the ```fcn``` model on the MTS dataset: Libras. 

Let me know if you need more help. 

Good luck !!

**Comment by neensimpson** [comment]
Hi!

Do we need to run any of the other functions in utils.py to read the data in and set up the appropriate folder structure?

I followed what you advised above, and got the following output running just transform_mts_to_ucr_format()

```
ArabicDigits max 93 min 4
AUSLAN max 136 min 45
CharacterTrajectories max 205 min 109 ...etc
```
but then when I issued the command for main and got the error
`[Errno 2] No such file or directory: '/mypath/dl-4-tsc/archives/mts_archive/Libras/x_train.npy'
`
So it seems like I may need to run transform_labels, create_directory, and read_dataset as well, to get all the folders set up. But all of those functions are imported into and used in main.py, so I don't understand why I'd need to do that. 

Can you explain a bit more what steps I should have followed after cloning the repository, downloading the datasets, and updating the root_dir in main.py and mts_root_dir and mts_out_dir in utils.py ?

**Comment by hfawaz** [comment]
Hi 

Since the run of transform_mts_to_ucr_format is successful, you should not have to use the create directory function. 

Can you check if the datasets are indeed created in mts_out_dir ?

**Comment by neensimpson** [comment]
Am I understanding this correctly? Running transform_mts_to_ucr should prepare all 13 .mat datasets so that they can be used by main, and it should do that by setting up an archive of .npy datasets in folders with matching names?

Unfortunately it wasn't creating the archive specified in mts_out_dir at all. I tried just making the folders archives and mts_archive manually but it still didn't create the dataset folders ('ArabicDigits' etc) so I manually created those too, and then ran transform_mts_to_ucr, removing the 'continue' on line 190 and it was able to make the x/y-test/train numpy files for each dataset.
